### PR TITLE
Fixed DeckGLMap python example and added test

### LINKED ISSUE #556: `DeckGLMap` python example does not work
The Deckgl python example in `examples/example_deckgl_map.py` does not render the map.
The Dash app successfully starts with no errors in shell or web console, but only an empty page is shown.

### PR DESCRIPTION
**Pull request by HansKallekleiv**
- [x] Added missing props to `DeckGLMap` python example, did some minor changes to layout and simplified imports
- [x] Added basic test for console errors

Closes #556

### CODE REVIEW

**Review by hkfb — APPROVED** [review]
Works fine now :+1:

**Review by anders-kiaer — APPROVED** [review]
Good job! :rocket: So the underlying issue was missing input data to the component? Should the component had given some console log errors in this case? And if so do we have an issue/task on getting some error from the component in case mandatory input data is missing?

**Comment by hkfb** [comment]
> Good job! So the underlying issue was missing input data to the component? Should the component had given some console log errors in this case? And if so do we have an issue/task on getting some error from the component in case mandatory input data is missing?

Created issue #562

File: setup.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
     .replace("@", "")
 )
 
-INSTALL_REQUIRES = ["dash>=1.6"]
+INSTALL_REQUIRES = ["dash>=1.6", "jsonpatch>=1.32"]
```
Comment (anders-kiaer) [inline setup.py:28]: Now I see what you mean @HansKallekleiv

**Review by anders-kiaer — CHANGES_REQUESTED** [review]

**Review by anders-kiaer — APPROVED** [review]

**Pull request merged**